docs/Import-the-Seed-Database.md

```diff
@@ -4,7 +4,7 @@ The next step is to populate your cBioPortal instance with all the required back
 
 ## Download the cBioPortal Database
 
-A cBioPortal seed database can be found on [the datahub page](https://github.com/cbioportal/datahub/blob/master/seedDB/README.md).
+A cBioPortal seed database for human can be found on [the datahub page](https://github.com/cBioPortal/datahub/blob/84fd66daf8325ad9721895d1cc503653686de15e/seedDB/README.md). If you are looking for mouse, check [this link](https://github.com/cBioPortal/datahub/blob/b02ac7037c8febeb86d6635602fc98327fb77c50/seedDB_mouse/README.md).
```

Comment:
I think releasing a new version of a dependency should not break the documentation and installation of existing releases of cBioPortal, still running at various institutions. However, @pieterlukasse suggests that we could also solve this by maintaining a list of links on the datahub `README.md` page itself: *that* page would then refer to specific commits' states of the sql files for specific releases of cBioPortal.